os/board/imxrt1050-evk/src/imxrt_boot.c

```diff
@@ -83,6 +84,64 @@
 #include "imxrt_semc_sdram.h"
 #endif
 
+#ifdef CONFIG_ANALOG
+#include "imxrt_adc.h"
+#endif
+
+
+
+/****************************************************************************
+ * Name: imxrt_board_adc_initialize
+ *
+ * Description:
+ *   Initialize the ADC. As the pins of each ADC channel are exported through
+ *   configurable GPIO and it is board-specific, information on available
+ *   ADC channels should be passed to imxrt_adc_initialize().
+ *
+ * Input Parameters:
+ *   chanlist  - The list of channels
+ *   cchannels - Number of channels
+ *
+ * Returned Value:
+ *   Valid ADC device structure reference on succcess; a NULL on failure
+ *
+ ****************************************************************************/
+void imxrt_board_adc_initialize(void)
+{
+	lldbg("%s IN\n", __FUNCTION__);
+#ifdef CONFIG_ANALOG
+	static bool adc_initialized = false;
```

Comment:
Let's include <stdbool.h> to use `bool`.

Reply:
Added.